tests/end-to-end/upgrade/pkg/tests/logging/pkg/logstream/logstream.go

```diff
@@ -103,3 +103,28 @@ func Cleanup(namespace string, coreInterface kubernetes.Interface) error {
 	}
 	return nil
 }
+
+func doGet(url string, authHeader string) (string, error) {
+	req, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		return "", errors.Wrap(err, "cannot create a new HTTP request")
+	}
+	req.Header.Add("Authorization", authHeader)
+	client := &http.Client{
```

Comment:
It is not a good idea to create a new client for every call. Can you just share it between calls?

Reply:
yes, I agree it is better to share the client.
The changes are pushed now